File: python/Hello.py
import HelloWorld
print "HelloWorld"

a=10
b=20
c=HelloWorld.plus(a,b)
print "{0}{1}".format("plus=",c)

sample = HelloWorld.Sample()
d=sample.minus(a,b)
print "{0}{1}".format("minus=",d)

File: golang/Hello.go
package main

import (
	"fmt" 
	"./HelloWorld" 
)

func main() {
    fmt.Println("HelloWorld")
    a:=10
    b:=20 
    c:=HelloWorld.Plus(a,b)
    fmt.Printf("plus=%d\n",c)
    d:=HelloWorld.Number{a,b}
    e:=d.Minus()
    fmt.Printf("minus=%d\n",e)
}

